packages/notifier-seeder/src/models/purpose/purposeEventNotification.ts

```diff
@@ -18,6 +18,7 @@ export type PurposeVersionV1Notification = Omit<
   | "updatedAt"
   | "firstActivationAt"
   | "suspendedAt"
+  | "expectedApprovalDate"
 > & {
   state: string;
   riskAnalysis?: PurposeVersionDocumentV1Notification;
```

Comment:
should we add it here?
```suggestion
  riskAnalysis?: PurposeVersionDocumentV1Notification;
  expectedApprovalDate?: string;
```

Reply:
Forwarding this question to @taglioni-r

Reply:
I think so. @Carminepo2 try to add that and also the conversion in `purposeEventNotificationMappers`.